client/components/screen-reader-text/style.scss

```diff
@@ -1,5 +1,5 @@
 // Text meant only for screen readers
-.screen-reader-text { // TODO add _accessibility.scss
```

Comment:
Completely unrelated, but I wanted to remove this todo.

It's confusing https://github.com/Automattic/jetpack/pull/12277#discussion_r282027675

And it's ancient: 1075-gh-calypso-pre-oss

I think it's safe to say that it's not a priority and no one is coming back to do it.